spec/tapioca/cli/annotations_spec.rb

```diff
@@ -22,7 +22,7 @@ class AnnotationsTest < SpecWithProject
 
         assert_stdout_equals(<<~OUT, result)
           Retrieving index from central repository... Done
-          Listing gems from Gemfile.lock... Done
+          Listing gems from bundle... Done
```

Comment:
```suggestion
          Listing gems from Gemfile.lock... Done
```

Can we add a test for `Gems.rb`, and comment that it's the old name of the file, supported for backwards compatibility?

Reply:
Added test.